translations/en.json

```diff
@@ -218,8 +218,8 @@
   "editor.record.form.field.onlineResource.fileSize": "{sizeMB}MB",
   "editor.record.form.field.onlineResource.modify": "Modify",
   "editor.record.form.field.overviews": "Overviews",
-  "editor.record.form.field.recordUpdated": "Record Updated",
-  "editor.record.form.field.resourceUpdated": "Resource Updated",
+  "editor.record.form.field.recordUpdated": "Dataset updated",
+  "editor.record.form.field.resourceUpdated": "Resource updated",
```

Comment:
I honestly don't get this diff. In technical terms, the record is the metadata and the resource is the data, but here you suggest the data for the record.

Reply:
Indeed. 
The notion of dataset encompasses data and their metadata.

See eg https://demo.ckan.org/dataset/warandpeace - the word "metadata" or "record" is nowhere to see.
See also https://stable.demo.geonode.org/catalogue/#/datasets?d=11161%3Bdataset%3Bvector 

We make the difference between data and metadata because we're fully immersed in the technical context.
But for most users, this is too complex to grasp. We need to lower the barrier to entry.

Reply:
OK. As discussed with @xxiiss , these metadata fields about the update date should not even be shown. We will remove those, and then your PR will be valid.